jobbergate-docs/pyproject.toml

```diff
@@ -1,23 +1,23 @@
 [tool.poetry]
 name = "jobbergate-documentation"
-version = "4.1.0a2"
+version = "4.1.0a3"
```

Comment:
This is unexpected, and got out of sync with the other components (same for changelog)

Reply:
This was leftovers from testing the version bump script.

Reply:
Fixed in commits 6d9a009 3619f47